proxy/reverse.go

```diff
@@ -56,6 +57,7 @@ type reverseProxy struct {
 func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request) {
 	proxyreq := new(http.Request)
 	*proxyreq = *clientreq
+	startTime := time.Now()
```

Comment:
I haven't seen any proxy code use `clockwork` and using `time` directly. @xiang90, what do you think?